app/assets/stylesheets/deploys.scss

```diff
@@ -61,6 +61,12 @@
   padding: 0 50px;
 }
 
+.deploy-search{
```

Comment:
this would be good to be more general for .search or whatever class we wrap around the search forms

Reply:
We don't have a general CSS for `search`, Moreover search_form for deploys have different design patterns compare to other searches.